### DISCUSSION TITLE
koyeb给我停掉了

### DISCUSSION THREAD

**Discussion by Mikurations** [luolongfei/freenom#181] [Q&A]
![Screenshot_20221216-074913_Chrome](https://user-images.githubusercontent.com/55773646/207990770-90654fdd-60d0-47f2-bb9b-64f30baf7865.png)
![Screenshot_20221215-205003_Gmail](https://user-images.githubusercontent.com/55773646/207990794-d4797a36-9387-4cba-8e7d-4dd40e2b7b89.png)

**Comment by luolongfei** [comment]
这没办法，免费的折腾麻烦
